examples/normal_sghmc.py

```diff
@@ -0,0 +1,80 @@
+#!/usr/bin/env python
+"""Correlated normal posterior. Inference with stochastic gradient Hamiltonian
+Monte Carlo.
+"""
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import edward as ed
+import tensorflow as tf
+import numpy as np
+from matplotlib import pyplot as plt
+plt.style.use("ggplot")
+
+from edward.models import Empirical, MultivariateNormalFull
+
+# Helper functions
+# TODO: Where should these go?
```

Comment:
I added some helper functions to visualize the contours of a bivariate Gaussian (or more generally, any Edward distribution with a pdf method). Should this be refactored?